src/lib/targeting.ts

```diff
@@ -46,12 +44,13 @@ export const shouldThrottle = (
     }
 
     const viewsInThrottleWindow = views.filter(view => {
-        return daysSince(new Date(view.date), now) <= config.maxViewsDays;
+        return daysSince(new Date(view.date), now) < config.maxViewsDays;
     });
 
-    const exceedsViewsInWindow = viewsInThrottleWindow.length >= config.maxViewsCount;
+    const exceedsViewsInWindow = viewsInThrottleWindow.length > config.maxViewsCount;
+
     const withinMinDaysSinceLastView = viewsInThrottleWindow.some(
-        view => daysSince(new Date(view.date), now) <= config.minDaysBetweenViews,
+        view => daysSince(new Date(view.date), now) < config.minDaysBetweenViews,
```

Comment:
Given `minDaysBetweenViews=1`...
Apply the throttle when `daysSince()` returns `0` (effectively less than 1 day since last view).